### PR TITLE
Pull Request template

### PR DESCRIPTION
**Pull request by AffirmedVisionary**
Not sure what else needs to be added. thinking it is one of those that will get more detailed as we go along maybe.

it is funny because I am not using the template right now and I am lost at what to write lol so it is definitely going to do its job.

fixes issue #3 

@tanyapowell can you jump on the reviewing of this too, just to make sure it has what is needed.

ps. the display rich diff button makes a real difference so try it out when reviewing the code

image

### CODE REVIEW

File: docs/PULL_REQUEST_TEMPLATE.md
Comment (tanyapowell) [inline docs/PULL_REQUEST_TEMPLATE.md:26]: Nitpick: there aren't "standard" `node_modules` so could "standard" be changed to _current_?

**Review by tanyapowell — COMMENTED** [review]
This is amazing 🎉! Great work @AffirmedVisionary. 
I won't approve it because I want the other contributors to look though and get more comfortable doing reviews 😄

File: docs/PULL_REQUEST_TEMPLATE.md
Comment (AffirmedVisionary) [reply docs/PULL_REQUEST_TEMPLATE.md:26]: okay, makes sense.

File: docs/PULL_REQUEST_TEMPLATE.md
Comment (AffirmedVisionary) [reply docs/PULL_REQUEST_TEMPLATE.md:35]: would that go here or in the contributing guidelines?

Comment (tanyapowell) [reply docs/PULL_REQUEST_TEMPLATE.md:35]: contributing guidelines

Comment (AffirmedVisionary) [reply docs/PULL_REQUEST_TEMPLATE.md:35]: cool @iesea if you could find space for this in the contributing guidelines #12 #5

Comment (iesea) [reply docs/PULL_REQUEST_TEMPLATE.md:35]: Hey yes, sorry to sound permeative, although I'll be happy to put these titles into the Guidelines Im unsure if you'd like the whole file to be copied over/ just the titles or a link? Right now I am creating a space for the titles above to go in but please make any changes you would like to see in the doc styling or wording so we have created a good good and clear for all who read it.

Comment (tanyapowell) [reply docs/PULL_REQUEST_TEMPLATE.md:35]: @AffirmedVisionary and @iesea could you work together to add this suggestion to #12, in a way that works best for the team and is easy for anyone new to jump on and understand 🥰

Comment (iesea) [reply docs/PULL_REQUEST_TEMPLATE.md:35]: Yes of course would love to look at this further - @AffirmedVisionary let’s work - let me know when you are free and I’ll meet you the same time to bang it out? But mainly to make sure users  can utilise the buttons in the way you vision because I’m unsure myself how to best use it where it clicks

Comment (AffirmedVisionary) [reply docs/PULL_REQUEST_TEMPLATE.md:35]: @iesea it is hard to keep on top of these reviews/comments in different places. I saw this after reviewing issue #12 and made a suggestion regarding this topic. We carry on conversations regarding this suggestion over there. x

Comment (AffirmedVisionary) [reply docs/PULL_REQUEST_TEMPLATE.md:35]: > Yes of course would love to look at this further - @AffirmedVisionary let’s work - let me know when you are free and I’ll meet you the same time to bang it out? But mainly to make sure users can utilise the buttons in the way you vision because I’m unsure myself how to best use it where it clicks

Which buttons?

**Comment by iesea** [comment]
The tick options sorry? Is the idea to tick them when using the template in
the doc/PULL... and also for when reading the guidelines - or use the info
from the template or signpost to the doc/PULL.. what do you think? If so I
currently can’t tick the boxes- so to make sure contributors can, are you
able to tick inside the boxes?

**Comment by AffirmedVisionary** [comment]
> The tick options sorry? Is the idea to tick them when using the template in the doc/PULL... and also for when reading the guidelines - or use the info from the template or signpost to the doc/PULL.. what do you think? If so I currently can’t tick the boxes- so to make sure contributors can, are you able to tick inside the boxes?

So to my understanding, the way the templates work is after this is approved and I have merged it to the main branch. 

Every time one of us create a pull request, this template will appear in the Write/Comment/Description section. We would then on our own go through and add to or edit each section with a ❓ next to it. if the boxes are not tickable, placing an x inside the square bracket works. you can check this in preview mode. <-- **you can add this to the guidelines.**

File: docs/PULL_REQUEST_TEMPLATE.md
Comment (EbonyChantelle) [inline docs/PULL_REQUEST_TEMPLATE.md:8]: Can we please have Breakdown of task, steps taken as I have OCD lol....

File: docs/PULL_REQUEST_TEMPLATE.md
Comment (EbonyChantelle) [inline docs/PULL_REQUEST_TEMPLATE.md:5]: Can we please have a , after change(s)?

File: docs/PULL_REQUEST_TEMPLATE.md
Comment (EbonyChantelle) [inline docs/PULL_REQUEST_TEMPLATE.md:20]: Can we please have a comma after changes, another after yes, and another after growth?

File: docs/PULL_REQUEST_TEMPLATE.md
Comment (EbonyChantelle) [inline docs/PULL_REQUEST_TEMPLATE.md:26]: Can we please have a comma after yes?

**Review by EbonyChantelle — COMMENTED** [review]
I know, I have serious OCD but you did say you were haunted without code @AffirmedVisionary lol....

**Comment by AffirmedVisionary** [comment]
> I know, I have serious OCD but you did say you were haunted without code @AffirmedVisionary lol....

lol all done x

File: docs/PULL_REQUEST_TEMPLATE.md
Comment (EbonyChantelle) [inline docs/PULL_REQUEST_TEMPLATE.md:20]: Just a ? after the first question and think we're good so shall approve.

File: docs/PULL_REQUEST_TEMPLATE.md
Comment (AffirmedVisionary) [inline docs/PULL_REQUEST_TEMPLATE.md:20]: @EbonyChantelle I re-worded it, check it out to see if it sounds better now.

Comment (EbonyChantelle) [reply docs/PULL_REQUEST_TEMPLATE.md:20]: Change to the first and last sentences slightly to the following and then we are good to go....

To complete the changes, did you use any tutorials or do any reading that would be of benefit to the group?

It is great for our growth, and can also aid with troubleshooting.

Comment (AffirmedVisionary) [reply docs/PULL_REQUEST_TEMPLATE.md:20]: okay

**Review by EbonyChantelle — APPROVED** [review]

**Pull request merged**
